### ISSUE TITLE
Latest release 1.1.0 breaks backwards compatibility

### ISSUE THREAD

**Issue by shusak** [arloliu/copy-node-modules#16]
Since the code switched to use `export`, this broke backwards compatibility for existing implementations. 

Now instead of `const copyNodeModules = require('copy-node-modules');` one needs to use
`const copyNodeModules = require('copy-node-modules').default;`

Why didn't the semver change to 2.x to indicate this breaking change?

**Comment by light24bulbs** [comment]
I have a feeling this effect was unintentional.  It's not in the examples..

**Comment by ezze** [comment]
@shusak, thanks for reporting this. @light24bulbs, yes, it was unintentional, sorry for that, guys.
I submitted a [commit](https://github.com/arloliu/copy-node-modules/commit/bd0bf5f2ba13c114e31eadfe0c1f998cf18380ed) that should fix backward compatibility. It should be possible to require a module by any of both methods:

```javascript
const copyNodeModules = require('copy-node-modules');
```

or

```javascript
const copyNodeModules = require('copy-node-modules').default;
```

@arloliu, can you check it and publish a fix to NPM?

**Comment by arloliu** [comment]
@ezze, sure, I had published v1.1.1 for this backward compatibility issue.
